build.sbt

```diff
@@ -14,41 +14,39 @@
  * limitations under the License.
  */
 
-import Dependencies._
-import JacocoSetup._
 import sbt.Keys.name
+import sbt.*
+import Dependencies.*
+import Dependencies.Versions.spark3
+import VersionAxes.*
 
 ThisBuild / organization := "za.co.absa.atum-service"
 sonatypeProfileName := "za.co.absa"
 
-ThisBuild / scalaVersion := Versions.scala213  // default version
+ThisBuild / scalaVersion := Setup.scala213  // default version
 
 ThisBuild / versionScheme := Some("early-semver")
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
 publish / skip := true
 
-lazy val printSparkScalaVersion = taskKey[Unit]("Print Spark and Scala versions for atum-service is being built for.")
-lazy val printScalaVersion = taskKey[Unit]("Print Scala versions for atum-service is being built for.")
+initialize := {
```

Comment:
Is the assignment necessary?

Reply:
To be honest I don't know. This code block is a remnant of my tries to keep Spakr 2.4 support. And I found it interesting, so kept it for now, with the intention to ask you what you think to have that information (of the active Java version) there.
Just forgot to comment accordingly.
So what do you think, should we keep the info (with the commented out cod removed)?